third_party/move/tools/move-cli/tests/build_tests/dependency_chain/args.exp

```diff
@@ -2,3 +2,9 @@ Command `build -v`:
 INCLUDING DEPENDENCY Bar
 INCLUDING DEPENDENCY Foo
 BUILDING A
+warning[W02016]: unknown attribute
+  ┌─ ./sources/A.move:1:3
+  │
+1 │ #[evm_contract] // for passing evm test flavor
```

Comment:
Should this be added to the evm flavor?

Reply:
#[evm_contract] is already in the evm flavor, but this test is run without evm flavor so it should yield a warning, according to my understanding.

I'm pretty sure we don't want any attributes to enable async/evm flavor if it's not selected by flag.  There is an arch= option in Move.toml files which in one test case selects "async" and I think it could select "evm", which will likely alter the compilation flavor, but since we're not supporting these flavors I don't think it's worth making sure it works.